std_msgs/Header header
pamaral_object_grasping_pattern_recognition/MpHand[] hands
pamaral_object_grasping_pattern_recognition/MpPose pose
